### PR TITLE
Push "latest" - point to 7.4

### PR DESCRIPTION
**Pull request by mvorisek**

### CODE REVIEW

File: .codefresh/deploy-build-image.yaml [hunk 1]
```diff
@@ -102,4 +102,9 @@ steps:
         candidate: "${{b8_0}}"
         type: push
         registry: atk4
-        tag: "8.0"
+        tag: "8.0""
```
Comment (romaninsh) [inline .codefresh/deploy-build-image.yaml]: ^ typo?

File: .codefresh/deploy-build-image.yaml [hunk 2]
```diff
@@ -102,4 +102,9 @@ steps:
...
+      platest:
```
Comment (romaninsh) [inline .codefresh/deploy-build-image.yaml:106]: latest or platest?

**Review by romaninsh — CHANGES_REQUESTED** [review]
some typos

File: .codefresh/deploy-build-image.yaml (hunk 1 above)
Comment (mvorisek) [reply .codefresh/deploy-build-image.yaml]:
```suggestion
        tag: "8.0"
```

Comment (mvorisek) [reply .codefresh/deploy-build-image.yaml]: yes, fixed

File: .codefresh/deploy-build-image.yaml (hunk 2 above)
Comment (mvorisek) [reply .codefresh/deploy-build-image.yaml:106]: it is name of the stage, so I think this is fine

**Review by romaninsh — APPROVED** [review]

**Pull request merged**